past-seasons.html

```diff
@@ -0,0 +1,271 @@
+                src="assets/images/past-seasons/2018/stemley.jpg"
+                class="event-image"
+              />
+            </div>
+            <div class="event">
+              <a
+                href="https://thebluealliance.com/event/2018onff"
+                class="event-title"
+              >
+                Fall Fiesta Junior
+              </a>
+              <h5 class="event-date">October 19, 2018</h5>
+              <h4 class="event-subtitle">Rank 15 & Quarterfinalist</h4>
+              <img
+                src="assets/images/past-seasons/2018/fall_fiesta.jpg"
+                class="event-image"
+              />
+            </div>
+            <div class="event">
+              <a
+                href="https://thebluealliance.com/event/2018dal"
+                class="event-title"
+              >
+                FIRST Robotics World Championship - Daly Division
+              </a>
+              <h5 class="event-date">April 25 - 28, 2018</h5>
+              <h4 class="event-subtitle">Rank 52</h4>
+              <img
+                src="assets/images/past-seasons/2018/worlds.jpg"
+                class="event-image"
+              />
+            </div>
+            <div class="event">
+              <a
+                href="https://thebluealliance.com/event/2018oncmp2"
+                class="event-title"
+              >
+                FIRST Ontario Provincial Championship - Technology Division
+              </a>
+              <h5 class="event-date">April 11 - 14, 2018</h5>
+              <h4 class="event-subtitle">Rank 20 & Finalist</h4>
+              <img
+                src="assets/images/past-seasons/2018/provs.jpg"
+                class="event-image"
+              />
+            </div>
+            <div class="event">
+              <a
+                href="https://thebluealliance.com/event/2018onham"
+                class="event-title"
+              >
+                ONT District - McMaster University Event
+              </a>
+              <h5 class="event-date">April 6 - 8, 2018</h5>
+              <h4 class="event-subtitle">Rank 26 & Finalist</h4>
+              <img
+                src="assets/images/past-seasons/2018/mcmaster.jpg"
+                class="event-image"
+              />
+            </div>
+            <div class="event">
+              <a
+                href="https://thebluealliance.com/event/2018onto1"
+                class="event-title"
+              >
+                ONT District - Ryerson University Event
+              </a>
+              <h5 class="event-date">March 15 - 17, 2018</h5>
+              <h4 class="event-subtitle">Rank 13 & Semifinalist</h4>
+              <img
+                src="assets/images/past-seasons/2018/ryerson.jpg"
+                class="event-image"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```

Comment:
Can this entire line be removed as it looks unprofessional.

Reply:
Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.